_includes/head--common.html

```diff
@@ -34,13 +34,12 @@
 	<link href="{{site.baseurl}}/assets/images/apple-touch-icon.png" rel="apple-touch-icon">
 
 
-	<script src="{{site.baseurl}}/assets/js/prism.js"></script>
```

Comment:
should only be loaded once in the footer. This also speeds up rendering the page